sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -224,6 +224,37 @@ def strip_unit_and_beautify(value, unit):
         return str(value)
 
 
+def notify_system_ready(fail_status=False, fail_reason='-'):
+    key = 'FEATURE|pmon'
+    statusvalue = {}
+
+    try:
+        state_db = swsscommon.SonicV2Connector(host='127.0.0.1')
```

Comment:
Why do you need to use a new connecter and state_db objects when you have those already defined in XcvrTableHelper? Please try to reuse them